wrapper.html

```diff
@@ -135,26 +136,34 @@
 	<meta name="twitter:site:id" content="{{ templateset.custom_fields.site_url }}" />
 
 	{% block meta_additions %}{% endblock %}
-<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+	<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>

```

Comment:
Did you mean to replace recaptcha with jQuery?

Reply:
I meant to remove Recaptcha based on Rachel's comment here: https://withtheranks.slack.com/archives/C029R3U6G8Z/p1635956112036500?thread_ts=1635954738.035000&cid=C029R3U6G8Z

Reply:
Copy that - the only thing I don't understand is why jquery was added in here. It seems like we are adding it to the page?